src/bin/www.ts

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env node
+
+/**
+ * Module dependencies.
+ */
+
+import * as http from "http";
+import * as config from "../config";
+import app from "../app";
+
+/**
+ * Get port from environment and store in Express.
+ */
+
+app.set("port", config.PORT);
+
+/**
+ * Create HTTP server.
+ */
+
+const server = http.createServer(app);
+
+/**
+ * Listen on provided port, on all network interfaces.
+ */
+
+server.listen(config.PORT);
+server.on("error", onError);
+
+/**
+ * Event listener for HTTP server "error" event.
+ */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+function onError(error: any) {
+  if (error.syscall !== "listen") {
+    throw error;
+  }
+
+  // handle specific listen errors with friendly messages
+  switch (error.code) {
+      case "EACCES":
+        // TODO implement logger
+        // logger.error(config.PORT + " requires elevated privileges");
```

Comment:
Should we add the logger integration as a new task to https://companieshouse.atlassian.net/browse/ROE-12 maybe?

Reply:
Morning, yes indeed